docker/Dockerfile.demo

```diff
@@ -21,13 +30,10 @@ COPY ./package.json .
 COPY ./yarn.lock .
 RUN yarn install
 
-COPY Pipfile .
-COPY Pipfile.lock .
+COPY requirements.txt .
```

Comment:
Does this migrate us out of Pipfile now?

Reply:
Yes, we're using `pip-tools` to generate a pinned set of dependencies just as `Pipfile.lock` did. The differences are that `pip-tools` doesn't lock the requirements to a specific version of Python, and also that `pip-tools` (according to docs at lesat) allows you to upgrade a single package instead of pipenv's behavior of always updating everything.

Reply:
Gotcha! As long as things are packaged as a Docker image, this shouldn't affect deployment too much!